src/pages/reservations/ReservationCard.js

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import './ReservationCard.css';
+import { removeReservation } from '../../api/reservationApi/reservation';
+import { clearReservation } from '../../slices/reservationSlice/reservationSlice';
+import { setAlert } from '../../slices/appSlice/appSlice';
+
+const ReservationCard = ({
+  id,
+  startTime,
+  endTime,
+  duration,
+  imageUrl,
+}) => {
+  const dispatch = useDispatch();
+  const [removing, setRemoving] = useState(false);
+  const token = useSelector((state) => state.auth.token);
+
+  const handleClick = () => {
+    setRemoving(true);
+    removeReservation(token, id).then((res) => {
+      if (res.status === 204) {
+        dispatch(clearReservation(id));
+      } else {
+        dispatch(setAlert('Canceling reservation failed!'));
+      }
```

Comment:
Optional: I think after successful clear, leave a successful message, cause it will look good(If you think otherwise please skip this)